scripts/deploy_docker.sh

```diff
@@ -2,15 +2,27 @@
 
 set -euo pipefail
 
-if [ "$1" = "develop" -o "$1" = "main" ]; then
-    # If image does not exist, don't use cache
-    docker pull gnosispm/$DOCKERHUB_PROJECT:$1 && \
-    docker build -t $DOCKERHUB_PROJECT -f Dockerfile . --cache-from gnosispm/$DOCKERHUB_PROJECT:$1 || \
-    docker build -t $DOCKERHUB_PROJECT -f Dockerfile .
-else
-    docker pull gnosispm/$DOCKERHUB_PROJECT:staging && \
-    docker build -t $DOCKERHUB_PROJECT -f Dockerfile . --cache-from gnosispm/$DOCKERHUB_PROJECT:staging || \
-    docker build -t $DOCKERHUB_PROJECT -f Dockerfile .
+if [ "$PULL_REQUEST_NUMBER" = "" ]; then
```

Comment:
Why do we need this?

Reply:
I just restored the old script, but I can get rid of it. I thought it was just double checking that this was comming from a PR.